core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -537,19 +539,17 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 							$trail[] = ' <span>' . $GLOBALS['TL_LANG']['MOD'][$table] . '</span>';
 						}
 
+						$strUrl = $container->get('router')->generate('contao_backend', array
+						(
+							'do' => $container->get('request_stack')->getCurrentRequest()->query->get('do'),
+							'table' => $table,
+							'id' => $objRow->id,
+							'ref' => $container->get('request_stack')->getCurrentRequest()->attributes->get('_contao_referer_id'),
```

Comment:
isnt the referrer ID supposed to change on every request?

Reply:
Yes, the request attribut contains the new one :)